.github/styles/UmbracoDocs/Acronyms.yml

```diff
@@ -83,3 +83,5 @@ exceptions:
     - SVG
     - DHL
     - UUI
+    - RCL
```

Comment:
I think it makes sense to add "MDN", as it sounds to me like this is the name now, and not an actual acronym.

But let's take out RCL here as well!
```suggestion
```

Reply:
Yes absolutely. MDN used to be "Mozilla Developer Network" or something to that effect, but now it has become their de facto name "MDN Web Docs".